lib/screenplay/alerts/parser.ex

```diff
@@ -96,19 +96,24 @@ defmodule Screenplay.Alerts.Parser do
 
   defp get_affected_list(%{"routes" => %{"data" => routes}}, included) do
     route_map =
-      Enum.map(included, fn %{"id" => id, "attributes" => %{"type" => route_type}} ->
-        {id, route_type}
+      Enum.map(included, fn
+        %{"id" => id, "attributes" => %{"short_name" => "SL" <> _}} ->
+          {id, "silver"}
+
+        %{"id" => id, "attributes" => %{"type" => route_type}} ->
+          {id, route_type}
       end)
       |> Enum.into(%{})
 
     affected_list =
       Enum.map(routes, fn %{"id" => id} ->
         case Map.get(route_map, id) do
           nil -> nil
+          route_type when route_type in [0, 1] -> String.downcase(id)
           2 -> "cr"
           3 -> "bus"
           4 -> "ferry"
-          _ -> String.downcase(id)
+          route_type -> String.downcase(route_type)
```

Comment:
Since this is a weird edge case for silver, perhaps add a "when" and/or a comment